js/core/visual.js

```diff
@@ -434,7 +434,11 @@ visual = {
    * @return {string} Either the string, or its first (len-3) characters and "...".
    */
   lengthLimit: function(str, len) {
-    return (len && str.length > len) ? str.substring(0, len-3) + '...' : str;
```

Comment:
Will change it to that, I checked out this patch on production and it does seem to allow messages to be displayed (although the "subject has been set" message could use some help :p)